ray-operator/api/raycluster/v1alpha1/raycluster_types.go

```diff
@@ -19,7 +19,8 @@ type RayClusterSpec struct {
 	// RayVersion is the version of ray being used. this affects the command used to start ray
 	RayVersion string `json:"rayVersion,omitempty"`
 	// EnableInTreeAutoscaling indicates whether operator should create in tree autoscaling configs
-	EnableInTreeAutoscaling *bool `json:"enableInTreeAutoscaling,omitempty"`
+	EnableInTreeAutoscaling *bool    `json:"enableInTreeAutoscaling,omitempty"`
+	DesiredWorkers          []string `json:"desiredWorkers,omitempty"`
```

Comment:
this would be  a full snapshot of the cluster workers? I remember community has some discussions in the past whether using full/delta earlier. Some feedbacks are if there's a large ray cluster, this list would be long.

Reply:
https://github.com/ray-project/kuberay/blob/659382250887608bce32abb94a49615581bef55e/ray-operator/api/raycluster/v1alpha1/raycluster_types.go#L57

We defined `ScaleStrategy` for this purpose in the past. Can that be reused?

Reply:
/cc @akanso

Reply:
In the current iteration, I'm now just launching worker pods directly in the gRPC server from the autoscaler. That's simple and effective and doesn't need schema changes.